File: engine/base/src/main/java/client/database/data/NoteData.java
package client.database.data;

public class NoteData {
   private int id;

   private String from;

   private String message;

   private long timestamp;

   private byte fame;

   public NoteData(int id, String from, String message, long timestamp, byte fame) {
      this.id = id;
      this.from = from;
      this.message = message;
      this.timestamp = timestamp;
      this.fame = fame;
   }

   public int getId() {
      return id;
   }

   public String getFrom() {
      return from;
   }

   public String getMessage() {
      return message;
   }

   public long getTimestamp() {
      return timestamp;
   }

   public byte getFame() {
      return fame;
   }
}

File: engine/engine-model/src/main/scala/client/database/data/NoteData.scala
package client.database.data

class NoteData(private var _id: Int, private var _from: String, private var _message: String,
               private var _timestamp: Long, private var _fame: Int) {
  def id: Int = _id

  def from: String = _from

  def message: String = _message

  def timestamp: Long = _timestamp

  def fame: Int = _fame
}
